tests/auth_tests/test_models.py

```diff
@@ -176,6 +177,23 @@ def test_last_login_default(self):
         user2 = User.objects.create_user(username='user2')
         self.assertIsNone(user2.last_login)
 
+    def test_user_clean_normalize_email(self):
+        """
+        Calling user.clean() will normalize email
```

Comment:
I think the test names and the tests themselves are straightforward enough that we can skip the docstrings.